src/service/platform-impl.js

```diff
@@ -158,6 +158,20 @@ export class Platform {
     version = version[1].replace(/_/g, '.');
     return version;
   }
+
+  /**
+   * Check whether current major IOS version is after specific version number.
+   * @param {number} version
+   * @return (boolean}
+   */
+  afterIosMajorVersion(version) {
```

Comment:
I'll use `getIosMajorVersion()` for this PR and use `compareVersion` for comparing ios minor versions in another PR.